engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscriptions-helper.service.js

```diff
@@ -49,7 +49,7 @@ angular.module('Bastion.subscriptions').service('SubscriptionsHelper',
 
             selected = [];
             angular.forEach(table.getSelected(), function (subscription) {
-                selected.push({"id": subscription.id});
+                selected.push({"id": subscription.cp_id});
```

Comment:
sorry for the delay.  I would  modify System#unsubscribe (and activation key) to check for nil (and error if it receives it), as i don't know we'd ever want to expose that functionality.  That way we won't accidentaly hit this again if we goof up the parameters calling the api